### DISCUSSION TITLE
Billing question - what does "one designer" mean

### DISCUSSION THREAD

**Discussion by RMBRIX** [primefaces/community#2348] [Figma UI Kit] [answered]
1. We have Figma for teams with one designer and multiple developers.  Does that mean we can get the single Designer ui kit?
2. When team grows, do we have to repurchase teams?

**Comment by onursenture** [comment, marked as answer]
Hi,

Yes, the number of designers matters for the UI Kit license. For a situation with one designer and multiple developers, you can get the "Single Designer" license.

If the number of designers in your team increases, you must upgrade your license to a Team license for up to 5 designers, or an Enterprise license if it exceeds five designers.

**Reply by RMBRIX** [reply]
Thank you for reply
